src/main/java/life/qbic/userdb/views/ProjectView.java

```diff
@@ -273,14 +267,9 @@ private String createTSVForTable() throws FileNotFoundException, UnsupportedEnco
       String name = proj.getSecondaryName();
       name = name == null ? "" : name;
       String space = proj.getSpace();
-      String inv = proj.getInvestigator();
-      inv = inv == null ? "" : inv;
-      String cont = proj.getContact();
-      cont = cont == null ? "" : cont;
-      String mang = proj.getManager();
-      mang = mang == null ? "" : mang;
       String row = replaceSpecialSymbols(
-          String.join("\t", Arrays.asList(code, name, space, inv, cont, mang)));
+          String.join("\t", Arrays.asList(code, name, space, getFullName(proj.getInvestigator()),
+              getFullName(proj.getContact()), getFullName(proj.getManager()))));
```

Comment:
Won't the UI now fail if a null value was provided for the manager, contact and investigator are provided? Previously this was avoided by the usage of the ternary operator for these values 🤔

Reply:
If a Person is null, getFullName returns an empty String.